docs/releasenotes/2.0.md

```diff
@@ -0,0 +1,245 @@
+
+See https://robotidy.readthedocs.io/en/latest/transformers/InlineIf.html for more examples.
+
+## Changes in transformers <a name="changes_in_transformer"></a>
+### `min_width` parameter <a name="min_width"></a>
+New ``min_width`` parameter was added to `AlignSettingsSection`, `AlignVariablesSection` and `AlignTestCases` ([#242](https://github.com/MarketSquare/robotframework-tidy/issues/242)).
+It allows to set minimal width of data column instead of using width of the widest argument in the column.
+
+Following code:
+
+    *** Variables ***
+    ${var}    a
+    ${var2}  b
+
+would be aligned to ``${var2}`` length since it's the longest token. If you wish to have fixed width you can use 
+``min_width`` parameter:
+
+```commandline
+robotidy --configure AlignVariablesSection:min_width=15
+```
+
+With such configuration previous code will be transformed to:
+
+    *** Variables ***
+    ${var}         a
+    ${var2}        b
+
+### Sections only with comments <a name="sections_only_with"></a>
+Previously sections that contained only comments and empty lines were removed by default by `DiscardEmptySections`. It 
+could be changed by configuring `allow_only_comments` to `True`. Since it wasn't expected behaviour by most users now it 
+works the opposite - now sections only with comments will be only removed by configuring `allow_only_comments` to `False` ([#235](https://github.com/MarketSquare/robotframework-tidy/issues/235)).
+
+
+### Do not align variables with given types <a name="skip_align"></a>
+New `skip_types` parameter in `AlignVariablesSection` which allows to not align variables of particular type ([#225](https://github.com/MarketSquare/robotframework-tidy/issues/225))
+
+With following configuration:
+
+```commandline
+    robotidy --configure AlignVariablesSection:skip_types=dict
+```
+
+All variables except dictionaries will be aligned.
+
+### `SplitTooLongLine` transformer <a name="split_too_long_line"></a>
+Several updates to `SplitTooLongLine` transformer.
+
+---
+
+Basic behaviour changed and now every statement that's longer than
+character limit will be split to multiple lines:
+
+```
+Keyword With Longer Name
+...    ${arg1}
+...    ${arg2}
+...    ${arg3}
+```
+
+If you wish to use previous algorithm of feeding arguments into one line until limit set ``split_on_every_arg`` to `False`:
+
+```
+Keyword With Longer Name    ${arg1}  # let's asumme that ${arg} is right below character limit
+...    ${arg2}    ${arg3}
+```
+
+---
+
+Assignments are now split (if they don't fit in one line):
+
+```
+${first_assignment}
+...    ${second_assignment}
+...    ${third_assignment}
+...    Some Lengthy Keyword So That This Line Is Too Long And Bit Over
+```
+
+### `RenameKeywords` transformer <a name="split_too_long_line"></a>
+Library names are now by default ignored by `RenameKeywords` transformer. Previously it could result in unexpected behaviour ([#269](https://github.com/MarketSquare/robotframework-tidy/issues/269)).
```

Comment:
```suggestion
Library names are now by default ignored by `RenameKeywords` transformer. Previously, it could result in an unexpected behaviour ([#269](https://github.com/MarketSquare/robotframework-tidy/issues/269)).
```